### PR TITLE
Responsive fixes, footer signup cleanups

### PR DESCRIPTION
**Pull request by alanmoo**
Addresses #234

### CODE REVIEW

**Comment by gvn** [comment]
You've got some linting errors ^

File: app/scss/main.scss [hunk 1]
```diff
@@ -54,6 +54,31 @@ h6 {
...
 
+.btn-knockout{
+  border-radius: 5px;
+  background: transparent;
+  border: 1px solid white;
+  color: white;
+  -webkit-filter: invert(0%);
+  filter: invert(0%);
+  background: transparent;
+  transition: all .75s ease;
```
Comment (gvn) [inline app/scss/main.scss]: This feels a little slow. Try 0.3-0.4

**Comment by alanmoo** [comment]
C’mon Sublime plugins, get your act together!

Fixed.

File: app/scss/main.scss (hunk 1 above)
Comment (gvn) [inline app/scss/main.scss]: Also, limit the transition to just the bg color.

File: app/scss/main.scss [hunk 2]
```diff
@@ -54,6 +54,31 @@ h6 {
   }
 }
 
+.btn-knockout{
+  border-radius: 5px;
+  background: transparent;
+  border: 1px solid white;
+  color: white;
+  -webkit-filter: invert(0%);
```
Comment (gvn) [inline app/scss/main.scss]: Does this not get added by autoprefixer?

File: app/scss/main.scss [hunk 3]
```diff
@@ -54,6 +54,31 @@ h6 {
...
+
+.btn-knockout:hover{
+    background: rgba(0,0,0,.5);
+}
+
+@supports (mix-blend-mode:screen){
+  .btn-knockout:hover{
+    border-color: #000;
+    background-color: #000;
+    -webkit-filter: invert(100%);
```
Comment (gvn) [inline app/scss/main.scss]: Autoprefixer?

**Comment by gvn** [comment]
See my "outdated comments": https://github.com/mozilla/science.mozilla.org/pull/429#discussion-diff-64286433

**Comment by gvn** [comment]
Minor nits then R+

File: app/scss/main.scss (hunk 2 above)
Comment (alanmoo) [inline app/scss/main.scss]: Realized it did while I was fixing linting, removed.

**Comment by alanmoo** [comment]
Good call on the timing, that was a debug time not meant to be final. Fixed and refactored the transition slightly to be easier to read.

**Comment by gvn** [comment]
👍

**Pull request merged**